### DISCUSSION TITLE
Support for Gitlab CI

### DISCUSSION THREAD

**Discussion by j4lc** [upptime/upptime#137] [Ideas]
Why not add the support of [Gitlab CI](https://docs.gitlab.com/ee/ci/) as the pipelines can be [scheduled as well](https://docs.gitlab.com/ee/ci/pipelines/schedules.html).
It would be a good idea for those who want to self-host that script or use it on a public Gitlab instance.

Edit: typos

**Comment by bigint** [comment]
Waiting for this 🤞🏻

**Comment by pcgeek86** [comment]
Would love to see GitLab support as well

**Comment by aparrot89** [comment]
Would love too
